### ISSUE TITLE
Who need calderon?

### ISSUE THREAD

**Issue by djcenox** [SkyFireArchives/SkyFireDB#63]
Quest of the bulkwark.(West. Plaguelands) I'm unable to put the Drums inside buildings.

**Comment by Bootz** [comment]
QuestId: http://www.wowhead.com/quest=26978
other related quest info:
http://www.wowpedia.org/Quest:Who_Needs_Cauldrons%3F (more research for quest)
http://www.wowhead.com/search?q=Who+need+cauldron%3F (related npc's)
http://www.wowhead.com/search?q=Who+need+cauldron%3F#spells (place drum spell)

untested by me just making some sense out of this report

**Comment by djcenox** [comment]
Yep sorry, the first one. It's better for me that next time to put a link i thought that everyquest was "unique" but seems not.
